apps/microfrontend-platform-testing-app/src/app/router-outlet-context/router-outlet-context.component.ts

```diff
@@ -100,11 +100,12 @@ export class RouterOutletContextComponent implements OnInit, OnDestroy {
     });
 
     const overlayRef = overlay.create(overlayConfig);
-    const injectionTokens = new WeakMap()
-      .set(OverlayRef, overlayRef)
-      .set(SciRouterOutletElement, routerOutlet);
+    const providers = [
+      {provide: OverlayRef, useValue: overlayRef},
+      {provide: SciRouterOutletElement, useValue: routerOutlet},
+    ];
 
-    const portal = new ComponentPortal(RouterOutletContextComponent, null, new PortalInjector(injector, injectionTokens));
+    const portal = new ComponentPortal(RouterOutletContextComponent, null, Injector.create({parent: injector, providers}));
```

Comment:
[IMO] Please inline the providers, as following:

```
Injector.create({
  parent: injector,
  providers: [
      {provide: OverlayRef, useValue: overlayRef},
      {provide: SciRouterOutletElement, useValue: routerOutlet},
  ],
});

```

Reply:
done